scripts/install_neovim.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+#
+# Install Neovim as /vim-build/bin/nvim.
+#
+# This is separate from install_vim.sh, because there is not much to share, but
+# uses the same names and tries to keep close to its ancestor.
+
+export CMAKE_EXTRA_FLAGS=-DENABLE_JEMALLOC=OFF
+NAME=nvim
+
+apk add --update-cache --no-cache --virtual neovim-build \
+  autoconf \
+  automake \
+  ca-certificates \
+  cmake \
+  curl \
+  g++ \
+  git \
+  libtool \
+  libuv \
+  m4 \
+  make \
+  ncurses ncurses-dev ncurses-libs ncurses-terminfo \
+  perl \
+  python3 \
+  python3-dev \
+  unzip \
+  xz \
+  && rm -rf /var/cache/apk/*
+
+git clone --depth=1 https://github.com/neovim/unibilium.git && \
+  cd unibilium && \
+  make && \
+  make install && \
+  cd ../ && rm -rf unibilium || exit 1
+
+git clone --depth=1 https://github.com/neovim/libtermkey.git && \
+  cd libtermkey && \
+  make && \
+  make install && \
+  cd ../ && rm -rf libtermkey || exit 1
+
+git clone --depth=1 https://github.com/neovim/libvterm.git && \
+  cd libvterm && \
+  make && \
+  make install && \
+  cd ../ && rm -rf libvterm || exit 1
+
+# NOTE: causes "bus error" with
+# % make CMAKE_EXTRA_FLAGS="-DCMAKE_INSTALL_PREFIX=$VIM_PATH"
+# > Scanning dependencies of target vimball-tags
+# > make[3]: Leaving directory '/neovim/build'
+# > make[3]: Entering directory '/neovim/build'
+# > Bus error (core dumped)
+# > runtime/CMakeFiles/vimball-tags.dir/build.make:57: recipe for target 'runtime/CMakeFiles/vimball-tags' failed
```

Comment:
Yeah, but it's reproducible.
Just tried with `make CMAKE_EXTRA_FLAGS="-DCMAKE_INSTALL_PREFIX=/nvim/nvim-master"` again.